# Portuguese translation for zeitgeist
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the zeitgeist package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: zeitgeist\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-11-01 08:28+0000\n"
"PO-Revision-Date: 2010-11-01 10:47+0000\n"
"Last-Translator: Tiago Silva <Unknown>\n"
"Language-Team: Portuguese <pt@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2010-11-02 04:53+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#: ../zeitgeist-daemon.py:68 ../zeitgeist-daemon.py:81
msgid "if another Zeitgeist instance is already running, replace it"
msgstr "se estiver outra instancia do Zeitgeist a correr, substitui-la"

#: ../zeitgeist-daemon.py:72
msgid "do not start zeitgeist-datahub automatically"
msgstr "não iniciar o zeitgeist-datahub automaticamente"

#: ../zeitgeist-daemon.py:76
msgid "how much information should be printed; possible values:"
msgstr "quanta informação deve ser impressa; valores possíveis:"

#~ msgid "Starting Zeitgeist service..."
#~ msgstr "A iniciar o serviço Zeitgeist..."

#, python-format
#~ msgid "Updating database with new %s items"
#~ msgstr "A actualizar base de dados com novos itens %s"

#~ msgid "Error: Zeitgeist service not running."
#~ msgstr "Erro: O serviço Zeitgeist não está em funcionamento."

#~ msgid "Could not connect to D-Bus."
#~ msgstr "Não foi possível conectar a D-Bus."

#~ msgid "Lost connection to zeitgeist-daemon, terminating."
#~ msgstr "Conexão com zeitgeist-daemon perdida. Terminando."

#, python-format
#~ msgid "Could not load file: %s"
#~ msgstr "Incapaz de ler o ficheiro: %s"

#~ msgid "Comments"
#~ msgstr "Comentários"

#~ msgid "Bookmarks"
#~ msgstr "Favoritos"

#~ msgid "Documents"
#~ msgstr "Documentos"

#, python-format
#~ msgid "Error logging item from \"%s\": %s"
#~ msgstr "Erro ao criar log do item de \"%s\": %s"

#~ msgid "Created"
#~ msgstr "Criado"

#~ msgid "Modified"
#~ msgstr "Modificado"

#~ msgid "Messages"
#~ msgstr "Mensagens"

#~ msgid "Feeds"
#~ msgstr "Fontes de notícias"

#~ msgid "Images"
#~ msgstr "Imagens"

#~ msgid "Music"
#~ msgstr "Música"

#~ msgid "Videos"
#~ msgstr "Vídeos"

#~ msgid "Visited"
#~ msgstr "Visitado"

#~ msgid "Activities"
#~ msgstr "Actividades"

#~ msgid "System Resources"
#~ msgstr "Recursos do Sistema"

#~ msgid "Notifications"
#~ msgstr "Notificações"

#~ msgid "Files"
#~ msgstr "Ficheiros"

#~ msgid "Received"
#~ msgstr "Recebido"

#~ msgid "Warnings"
#~ msgstr "Avisos"

#~ msgid "Send"
#~ msgstr "Enviar"

#~ msgid "Web History"
#~ msgstr "Histórico Web"

#~ msgid "Errors"
#~ msgstr "Erros"

#~ msgid "Applications"
#~ msgstr "Aplicações"

#~ msgid "Websites visited with Firefox"
#~ msgstr "Sítios visitados com Firefox"

#~ msgid "Firefox History"
#~ msgstr "Histórico do Firefox"

#~ msgid "No Firefox profile found."
#~ msgstr "Nenhum perfil do Firefox encontrado."

#~ msgid "Could not import GTK; data source disabled."
#~ msgstr "Incapaz de importar GTK; fonte dos dados desabilitada."

#, python-format
#~ msgid "Watching for Tomboy notes in \"%s\"."
#~ msgstr "A procurar por notas do Tomboy em \"%s\"."

#, python-format
#~ msgid "Monitoring Evolution: %s"
#~ msgstr "A monitorar Evolution: %s"

#, python-format
#~ msgid "Unable to monitor Firefox's history: %s: %s"
#~ msgstr "Incapaz de monitorar o histórico do Firefox: %s: %s"

#, python-format
#~ msgid "Unable to monitor Evolution: %s: %s"
#~ msgstr "Incapaz de monitorar o Evolution: %s: %s"

#, python-format
#~ msgid "Monitoring Firefox's history: %s"
#~ msgstr "A monitorar o histórico do Firefox: %s"

#~ msgid "Tags"
#~ msgstr "Etiquetas"

#~ msgid "Broadcasts"
#~ msgstr "Difusões"

#~ msgid "No passive loggers found, bye."
#~ msgstr "Nenhum logger passivo encontrado, adeus."

#~ msgid "Email"
#~ msgstr "Correio Electrónico"

#, python-format
#~ msgid "File \"%s\" not found, not starting datahub"
#~ msgstr "O ficheiro \"%s\" não foi encontrado, datahub não irá iniciar"
